homeassistant/components/group/__init__.py

```diff
@@ -183,7 +187,10 @@ def expand_entity_ids(hass: HomeAssistant, entity_ids: Iterable[Any]) -> list[st
 
         entity_id = entity_id.lower()
         # If entity_id points at a group, expand it
-        if entity_id.startswith(ENTITY_PREFIX):
+        if entity_id.startswith(ENTITY_PREFIX) or (
+            (source := entity_sources(hass).get(entity_id))
+            and source["domain"] == "group"
```

Comment:
We would bypass the service handlers in the platforms of the group integration if we'd do this. Eg:

https://github.com/home-assistant/core/blob/dd3790641aa349b66012485083d02b51c8f2640d/homeassistant/components/group/cover.py#L179-L184

You need to analyze the problem further and create an architecture discussion if you want to change this, as it would be a major change.

https://github.com/home-assistant/architecture/discussions/